src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java

```diff
@@ -1735,6 +1737,18 @@ public void siControllaLoStatoTimelineInDettaglioNotifica(Map<String, String> da
         dettaglioNotificaMittenteSection.checkStatoTimeline(idStato);
     }
 
+    @And("Si verifica la cliccabilità della ricevuta di postalizzazione")
+    public void siVerificaLaCliccabilitaDellaRicevutaDiPostalizzazione(Map<String, String> datiDettaglioNotifica) {
```

Comment:
I would not have created a new method but i would inserted another parameter in Si controlla lo stato timeline in dettaglio notifica which check the xpath you have passed is clickable